public/root.html

```diff
@@ -0,0 +1,5 @@
+<div ng-include="'header.html'"></div>
```

Comment:
I gotta use the resolve from the root state so there's no delay, I can fix that here or in another branch